eng/build.ps1

```diff
@@ -7,9 +7,12 @@ Param(
   [switch] $ci,
   [switch] $skipmanaged,
   [switch] $skipnative,
-  [switch] $dailytest,
   [string] $privatebuildpath = "",
   [switch] $cleanupprivatebuild,
+  [ValidatePattern("(\d+\.\d+.\d+(-[a-z0-9\.]+)?)?")][string] $dotnetruntimeversion = '',
+  [ValidatePattern("(\d+\.\d+.\d+(-[a-z0-9\.]+)?)?")][string] $dotnetruntimedownloadversion= '',
```

Comment:
This accepts either blank or a version. I don't think it accepts the word default.